test/e2e/resize_nodes.go

```diff
@@ -372,18 +410,19 @@ var _ = Describe("Nodes", func() {
 					break
 				}
 			}
-			Logf("Setting network partition on %s", node.Name)
+			By(fmt.Sprintf("cause network partition on node %s", node.Name))
+
 			dropCmd := fmt.Sprintf("sudo iptables -I OUTPUT 1 -d %s -j DROP", testContext.CloudConfig.MasterName)
 			if _, _, code, err := SSH(dropCmd, host, testContext.Provider); code != 0 || err != nil {
 				Failf("Expected 0 exit code and nil error when running %s on %s, got %d and %v",
-					dropCmd, node, code, err)
+					dropCmd, node.Name, code, err)
 			}
```

Comment:
Please move the reattachement code below (lines 431-436) into a defer block; Otherwise, if the test fails before that, the node will be permanently broken and future tests will probably fail.